### DISCUSSION TITLE
Is it possible to make a minio cluster of two standalone minio servers?

### DISCUSSION THREAD

**Discussion by shoce** [minio/minio#17889] [Q&A] [answered]
So I had a standalone minio server running in docker. After adding another disk to the server I found out I cannot just add the disk to the existing minio instance - it is not supported. So I started another minio instance in docker with the second disk connected. Now I have two minio standalone server instances running on the same server in docker each with it's own disk. Now can I run another minio server instance joining the two standalone instances into a cluster so I have total storage of the both? 

What is the best close-to-zero-downtime way to expand my minio storage on the same one server with adding disks only? I am thinking how to do it the next time when I need more disks.

**Comment by harshavardhana** [comment, marked as answer]
No it is not possible
